ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -81,6 +81,32 @@ AHU_CSP_DFSS_DXZC:
   - DFSS
   - DXZC
 
+AHU_SS_STC_RTM:
```

Comment:
does not conform to AHU def. please adjust.

should have outside air, return air, and supply or discharge fan control, plus conditioning.

Reply:
This is a crac unit with built in controls and only few points are shared to BOS via software interface for monitoring like supply fan, compressor points, alarms, supply air temperature and return air temperature and relative humidity. There are no command/setpoint points shared for monitoring which are internal.

Reply:
does it handle outside air? if not, it is an FCU. Regardless of what data is available to BOS, it should still be modeled around how it actually controls and you should use MISSING in the translation (see [here](https://github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md#defining-translations)).

Reply:
Modeled as FCU